docs/usage/mount.rst.inc

```diff
@@ -38,3 +38,10 @@ This command mounts an archive as a FUSE filesystem. This can be useful for
 browsing an archive or restoring individual files. Unless the ``--foreground``
 option is given the command will run in the background until the filesystem
 is ``umounted``.
+
+The command ``borgfs`` provides a wrapper for ``borg mount``. This can also be
+used in fstab entries:
+``/path/to/repo::archive /mnt/point fuse.borgfs defaults,noauto 0 0``
+
+To allow a regular user to use fstab entries, add the ``user`` option:
+``/path/to/repo::archive /mnt/point fuse.borgfs defaults,noauto,user 0 0``
```

Comment:
btw, did you produce this manually? there is a build_usage target that does create the .inc files automatically.